packages/components/src/DatePicker/DatePicker.mdx

```diff
@@ -99,6 +99,31 @@ pass a `React` component as the `activator`.
   }}
 </Playground>
 
+## Disabling dates
```

Comment:
I think it would be more accurate to call this section "Restricting date ranges", as disabling the actions is just a means to enforce the range restriction.